### DISCUSSION TITLE
How to make Enhanced Workflow the default and the only possible choice?

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#534] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

13

### Description

Hello guys!

I am wondering if there is a way to configure the company such that all the projects are with Enhanced Workflow by default and to disable the  possibility for the user to change this choice. 

I have a custom orchestrator generator and all the projects of my company must use it.

Thanks in advnace!

**Comment by akelity** [comment, marked as answer]
Hi,
Starting from version 14, the enhanced workflow will be introduced as the sole system for creating all new projects. 
This should address your specific need.
